lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -121,47 +125,87 @@ class IntegrationTestsManager {
     final List<String> blockedTests =
         blockedTestsListsMap[getBlockedTestsListMapKey(_browser)] ?? <String>[];
 
-    // The following loops over the contents of the directory and saves an
-    // expected driver file name for each e2e test assuming any dart file
-    // not ending with `_test.dart` is an e2e test.
-    // Other files are not considered since developers can add files such as
-    // README.
-    for (io.File f in entities) {
-      final String basename = pathlib.basename(f.path);
-      if (!basename.contains('_test.dart') && basename.endsWith('.dart')) {
-        // Do not add the basename if it is in the `blockedTests`.
-        if (!blockedTests.contains(basename)) {
-          e2eTestsToRun.add(basename);
-        } else {
-          print('INFO: Test $basename is skipped since it is blocked for '
-              '${getBlockedTestsListMapKey(_browser)}');
+    // If no target is specified run all the tests.
+    if (_runAllTestTargets) {
+      // The following loops over the contents of the directory and saves an
+      // expected driver file name for each e2e test assuming any dart file
+      // not ending with `_test.dart` is an e2e test.
+      // Other files are not considered since developers can add files such as
+      // README.
+      for (io.File f in entities) {
+        final String basename = pathlib.basename(f.path);
+        if (!basename.contains('_test.dart') && basename.endsWith('.dart')) {
+          // Do not add the basename if it is in the `blockedTests`.
+          if (!blockedTests.contains(basename)) {
+            e2eTestsToRun.add(basename);
+          } else {
+            print('INFO: Test $basename is skipped since it is blocked for '
+                '${getBlockedTestsListMapKey(_browser)}');
+          }
         }
       }
+      print(
+          'INFO: In project ${directory} ${e2eTestsToRun.length} tests to run.');
+    } else {
+      // If a target is specified it will run regardless of if it's blocked or
+      // not. There will be an info note to warn the developer.
+      final String targetTest =
+          IntegrationTestsArgumentParser.instance.testTarget;
+      final io.File file =
+          entities.singleWhere((f) => pathlib.basename(f.path) == targetTest);
+      final String basename = pathlib.basename(file.path);
+      if (blockedTests.contains(basename)) {
+        print('INFO: Test $basename do not run on CI environments.');
```

Comment:
I'd say "Skipping test $basename" (for consistency with `flutter test` and `pub run test`), unless I'm misunderstanding what blocked test means.

Reply:
Felt will run the test. 

The warning is for letting developer know that CI environment do not run the test. I added more information to the warning.